content/components/action-bar.mdx

```diff
@@ -168,6 +208,13 @@ Action bars can have 3 different sizes:
 - Medium (`32px`) (default)
 - Large (`40px`)
 
+### Button variants
+
+Icon buttons should default to the `invisible` (no border/background) variant. Buttons with text labels should default to the `secondary` variant.
+
+You may use the `danger` variant for destructive actions. The `primary` variant should not be used in an action bar because primary page actions are highly unlikely to appear in a collection with other actions.
+
```

Comment:
The API could be
- ActionBar.IconButton (default to invisible, cannot accept text)
- ActionBar.TextButton (default to secondary, can accept icon?)
- ActionBar.DangerButton (default to danger, can accept icon?)

Reply:
I think we might be able to have a single component to handle all three. An IconButton will need text _somewhere_ for screen readers and tooltips.

Reply:
If we want to be consistent with the Button API, we could keep `ActionBar.IconButton` separate

Reply:
`IconButton` does take `aria-label` which I'm currently using for the text when I pop it into the menu on overflow.
But I'm just torn between allowing variants broadly on `ActionBar.Button` or make different components. Let me think more on it and consult.

Reply:
> If we want to be consistent with the Button API, we could keep ActionBar.IconButton separate

+1 to that. I personally like the idea of keeping the names consistent with the Button API

1) `<ActionBar.IconButton icon={PaperclipIcon} aria-label="Attach text" />`
2) `<ActionBar.Button leadingVisual={LabelIcon}>Set Label</ActionBar.Button>`
3) `<ActionBar.Button variant="danger" leadingVisual={LabelIcon}>Set Label</ActionBar.Button>`

We could, of course, narrow down the props in ActionBar.Buttons to only support a subset of the Button props. For example, 
1) a limited set of variants instead of the whole set
2) only supporting leadingVisual and not trailingVisual or count. (bonus: a shortcut prop called `icon` instead of `leadingVisual`)
3) allow `size` on ActionBar, but not on ActionBar.Buttons

Reply:
Yes! I am fine with this reco thanks!